Game_Manual/Boundaries.md

```diff
@@ -0,0 +1,43 @@
+# Game Boundaries
+
+The boundaries of the game define its edges: what the objective is, and how a player succeeds or fails.
+
+It is important to note that the game is _not_ zero-sum: one player winning does not mean another loses. It is a multiplayer cooperative game where each player's success is interconnected to and supported by other players' successes.
+
+## Objectives
+
+The game is designed to support players in tracking and achieving two objectives:
+
+1. To become a world class beginner in their craft
+1. To co-create a culture of collective learning
+
+## Success Conditions
+
+A player succeeds when they (a) meet the above objectives and (b) fulfill the obligations in their contract.
+
+At this point, they are ready to pursue their craft as a practitioner and no longer need the support and scaffolding the game provides.
+
+## Fail Conditions
+
+A player fails the game if their [stats][stats] show them to be consistently underperforming.
+
+Fail conditions are triggered when any of the following persists over multiple cycles:
+
+1. [Time Dedication per Cycle][time-dedication] is too low
+1. Rate of [XP][xp] growth is too low
+1. [Learning Support][learning-support] is too low
+1. [Culture Contribution][culture-contribution] is too low
+1. [Discernment][discernment] is too high or too low
+
+If these conditions are met, the Moderator will expel the player from the game.
```

Comment:
I can't figure out what word is best for this. I don't like "kick out" (kinda physically violent) or "expel" (too school-related), but I prefer expel for now.